{
  "uid" : "4d8f2691543f9ec3",
  "name" : "全部订单测试",
  "fullName" : "TestCases.test_customer_orders.TestCustomerOrders#test_order_list",
  "historyId" : "19a96a711478aff2f3d693c2e0f9bc48",
  "time" : {
    "start" : 1749206167070,
    "stop" : 1749206167070,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "AttributeError: 'TestCustomerOrders' object has no attribute 'logger'",
  "statusTrace" : "self = <test_customer_orders.TestCustomerOrders object at 0x0000026984160C40>\n\n    def setup_method(self):\n        \"\"\"在fixture执行后，开始测试方法前记录日志\"\"\"\n>       self.logger.info(f\"------------------ 开始执行测试用例 ------------------\")\nE       AttributeError: 'TestCustomerOrders' object has no attribute 'logger'\n\nTestCases\\test_customer_orders.py:26: AttributeError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "_xunit_setup_method_fixture_TestCustomerOrders",
    "time" : {
      "start" : 1749206167071,
      "stop" : 1749206167071,
      "duration" : 0
    },
    "status" : "broken",
    "statusMessage" : "AttributeError: 'TestCustomerOrders' object has no attribute 'logger'\n",
    "statusTrace" : "  File \"D:\\python_tools\\python\\lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"D:\\python_tools\\python\\lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"D:\\python_tools\\python\\lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\python_tools\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\python_tools\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"D:\\python_tools\\python\\lib\\site-packages\\_pytest\\python.py\", line 821, in xunit_setup_method_fixture\n    _call_with_optional_argument(func, method)\n  File \"D:\\python_tools\\python\\lib\\site-packages\\_pytest\\python.py\", line 717, in _call_with_optional_argument\n    func()\n  File \"D:\\pycharm_test\\api_test_framework_refactor\\TestCases\\test_customer_orders.py\", line 26, in setup_method\n    self.logger.info(f\"------------------ 开始执行测试用例 ------------------\")\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : true,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true
  } ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "epic",
    "value" : "客服后台-全部订单-查询校验"
  }, {
    "name" : "parentSuite",
    "value" : "TestCases"
  }, {
    "name" : "suite",
    "value" : "test_customer_orders"
  }, {
    "name" : "subSuite",
    "value" : "TestCustomerOrders"
  }, {
    "name" : "host",
    "value" : "DESKTOP-4S9CU1E"
  }, {
    "name" : "thread",
    "value" : "1656-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "TestCases.test_customer_orders"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "4d8f2691543f9ec3.json",
  "parameterValues" : [ ]
}